src/routes/project/controllers/getProjects.ts

```diff
@@ -0,0 +1,41 @@
+import { Context } from 'koa';
+import { Types } from 'mongoose';
+import Project from '../../../models/project';
+
+enum UserType {
+  OWNER = 'owner',
+  MEMBER = 'member',
+  ALL = 'all',
+}
+
+interface IQuery {
+  userType: UserType;
+}
+export default async (ctx: Context): Promise<void> => {
+  const { userType = UserType.ALL }: IQuery = ctx.query;
+  const user = {
+    id: '5fc055d7d58baa2b2807974b',
+    projects: [
+      { _id: '5fc6026191611635d8983356' },
+      { _id: '5fc6026191611635d8983358' },
+      { _id: '5fc6026191611635d898335a' },
+    ],
+  };
+  try {
+    let projects = await Project.find()
+      .where('_id')
+      .in(user.projects.map((project) => Types.ObjectId(project._id)));
+    if (userType === UserType.OWNER) {
+      projects = projects.filter(({ owner }) => owner === user.id);
+    }
+    if (userType === UserType.MEMBER) {
+      projects = projects.filter(({ users }) => {
+        return !!users.find((id) => id === user.id);
+      });
+    }
```

Comment:
```suggestion
    if (userType === UserType.MEMBER) {
      projects = projects.filter(({ users }) => {
        return users.some((id) => id === user.id);
      });
    }
```
[Array.some() 함수](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/some)를 쓴다면 더 좋을 것 같습니다 😁

Reply:
깜빡 잊고 있었네요.. 감사합니다!! some을 사용하는게 훨씬 낫겠네요..